src/main/preload/externalAPI.ts

```diff
@@ -72,8 +73,8 @@ const desktopAPI: DesktopAPI = {
     setSessionExpired: (isExpired) => ipcRenderer.send(SESSION_EXPIRED, isExpired),
     onUserActivityUpdate: (listener) => createListener(USER_ACTIVITY_UPDATE, listener),
 
-    onLogin: () => ipcRenderer.send(APP_LOGGED_IN),
-    onLogout: () => ipcRenderer.send(APP_LOGGED_OUT),
+    onLogin: () => ipcRenderer.send(TAB_LOGGED_IN, true),
+    onLogout: () => ipcRenderer.send(TAB_LOGGED_IN, false),
```

Comment:
Nitpicking, but this naming is a bit confusing because TAB_LOGGED_IN is sent when you log out. I'd suggest something like TAB_LOGIN_CHANGED if you think it matters enough to change

Reply:
I just wanted it to be distinct from the other ones and I think just one name works. But you name is better I'll change it real quick.